modules/nf-core/vcflib/vcffixup/main.nf

```diff
@@ -0,0 +1,46 @@
+process VCFLIB_VCFFIXUP {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vcflib:1.0.3--hecb563c_1':
+        'biocontainers/vcflib:1.0.3--hecb563c_1' }"
+
+    input:
+    tuple val(meta), path(vcf), path(tbi)
+
+    output:
+    tuple val(meta), path("*.vcf.gz"), emit: vcf
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '1.0.3' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+
+    """
+    vcffixup \\
```

Comment:
No `args` for the tool?

Reply:
It does not have any optional `args` (https://github.com/vcflib/vcflib/blob/master/doc/vcffixup.md), unlike [other tools ](https://github.com/vcflib/vcflib/blob/master/doc/vcfroc.md) from the same suite. Would you suggest still adding them anyway?